src/lib/undux/SimpleStore.js

```diff
@@ -107,6 +107,7 @@ const initStore = async () => {
   let isLoggedIn = await AsyncStorage.getItem(IS_LOGGED_IN).then(JSON.parse)
   initialState.isLoggedIn = isLoggedIn
   SimpleStore = createConnectedStore(initialState, withPinoLogger)
+  global.SimpleStore = SimpleStore
```

Comment:
do we have to expose store to the global ns due to the some reasons (e.g. for automated tests) ?
if yes - i think we should check which env we're running and do it only for the dev/qa

Reply:
created separate task about this https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/1827